src/browzine-summon-adapter.js

```diff
@@ -98,56 +124,103 @@ angular.module("summonApp.directives")
+      wording = browzine.articleWording || "View Complete Issue";
+      browzineWebLinkText = browzine.articleBrowZineWebLinkText || "Browse Now";
     }
 
-    // You can change the underlined "Browse Now" link name on line 122 below.
     var template = "<div class='browzine'>" +
-                     "{{assetClass}}: <a class='browzine-web-link' href='{{browzineWebLink}}' target='_blank' style='text-decoration: underline; color: #333;'>Browse Now</a> " +
+                     "{{wording}}: <a class='browzine-web-link' href='{{browzineWebLink}}' target='_blank' style='text-decoration: underline; color: #333;'>{{browzineWebLinkText}}</a> " +
                      "<img class='browzine-book-icon' src='{{bookIcon}}'/>" +
                    "</div>";
 
-    template = template.replace(/{{assetClass}}/g, assetClass);
+    template = template.replace(/{{wording}}/g, wording);
     template = template.replace(/{{browzineWebLink}}/g, browzineWebLink);
+    template = template.replace(/{{browzineWebLinkText}}/g, browzineWebLinkText);
     template = template.replace(/{{bookIcon}}/g, bookIcon);
 
     return template;
   };
 
-  return {
-    link: function(scope, element, attributes) {
-      if(!shouldEnhance(scope)) {
-        return;
-      }
+  function getScope(documentSummary) {
+    return angular.element(documentSummary).scope();
+  };
 
-      var endpoint = getEndpoint(scope);
+  function resultsWithBrowZine(documentSummary) {
+    var scope = getScope(documentSummary);
 
-      http.get(sce.trustAsResourceUrl(endpoint)).then(function(response) {
-        var data = getData(response);
-        var browzineWebLink = getBrowZineWebLink(data);
-        var coverImageUrl = getCoverImageUrl(data, response);
+    if(!shouldEnhance(scope)) {
+      return;
+    }
 
-        if(browzineWebLink) {
-          var template = buildTemplate(data, browzineWebLink, bookIcon);
-          element.find(".docFooter .row:first").append(template);
-        }
+    var endpoint = getEndpoint(scope);
 
-        if(coverImageUrl) {
-          element.find(".coverImage img").attr("src", coverImageUrl);
-        }
-      });
-    }
+    $.getJSON(endpoint, function(response) {
+      var data = getData(response);
+      var browzineWebLink = getBrowZineWebLink(data);
+      var coverImageUrl = getCoverImageUrl(data, response);
+
+      if(browzineWebLink) {
+        var template = buildTemplate(data, browzineWebLink);
+        $(documentSummary).find(".docFooter .row:eq(0)").append(template);
+      }
+
+      if(coverImageUrl) {
+        $(documentSummary).find(".coverImage img").attr("src", coverImageUrl);
+      }
+    });
+  };
+
+  return {
+    resultsWithBrowZine: resultsWithBrowZine,
+    getScope: getScope,
+    shouldEnhance: shouldEnhance,
+    getEndpoint: getEndpoint,
+    getData: getData,
+    getBrowZineWebLink: getBrowZineWebLink,
+    getCoverImageUrl: getCoverImageUrl,
+    buildTemplate: buildTemplate,
+  };
```

Comment:
This is a much cleaner and more readable end return statement!

Reply:
Yep! Almost went w/ Rebbeca Murphy's [Object Literal Pattern](http://rmurphey.com/blog/2009/10/15/using-objects-to-organize-your-code), but went w/ the Revealing Module Pattern here for it's ability to have both private and public properties/methods. Combining that w/ the `browzine` namespace really helped to expose units of code to test while not polluting the global namespace. It's amazing how code bases evolve.

Reply:
If we had the ES6 features node version 6 has, or babel provides on Ember, we could even have the wildly clean
```
return {
  resultsWithBrowZine,
  shouldEnhance,
  getEndpoint,
  getData,
  getBrowZineWebLink,
  getCoverImageUrl,
  buildTemplate,
};
```

Just thought I'd mention that in case anyone had not seen this before - it's a nice little cleanup, though doubt we can do it and keep compatible with all the browsers we currently work with.